views/apartments/id_tenants_new.erb

```diff
@@ -0,0 +1,8 @@
+<form action="/new" method="post">
```

Comment:
action should be `/artists`. The creation is implied because you're using the POST http verb